### ISSUE TITLE
Issue with choices that are translations

### ISSUE THREAD

**Issue by jole78** [graphql-python/graphene-django#17]
I have a model that looks something like this: 

```python
class Organisation(models.Model):

	CATEGORY_CHOICES = (
		(1, _('Government administrative authorities')),
		(2, _('Authorities under parliament')),
		(3, _('State utilities')),
		(4, _('AP fonds')),
		(5, _('Courts of Sweden and courts administration')),
		(6, _('Swedish foreign authorities')),
	)
	category = models.IntegerField(_('category'), 
		choices=CATEGORY_CHOICES
	)
```

The import part is that category is a choice that has a translation (which is in Swedish btw).
When I query for this field I get this (as an example):
"category": "STATLIGA_F_RVALTNINGSMYNDIGHETER"

...which looks pretty weird to me.

The schema looks like this:
```python
class OrganisationNode(DjangoObjectType):

	class Meta:
		model = Organisation
		interfaces = (graphene.Node,)
```
 
If I instead do this:
```python
 class OrganisationNode(DjangoObjectType):
	
	category = graphene.String()

	class Meta:
		model = Organisation
		interfaces = (graphene.Node,)
	
	def resolve_category(self, args, context, info):
		return self.get_category_display()
```
...the result is as expected, e.g.:
"category": "Statliga förvaltningsmyndigheter"

So, there is a way around it. I was just wondering if this is by design or...??

**Comment by syrusakbary** [comment]
This should be fixed with #23, as it involve mapping the enum name to the choice value instead of the description.

**Comment by morinx** [comment]
@syrusakbary Is there an easy way to still access the description in the Node?
